_source/_posts/2021-08-05-libsodium-encryption-go-python.md

```diff
@@ -0,0 +1,340 @@
+---
+layout: blog_post
+title: Getting Started with Libsodium in Python and Go
+author: phill-edwards
+by: contractor
+communities: [python,go]
+description: "Tutorial: Learn how to use libsodium to encrypt files in Python and decrypt them in Go!"
+tags: [python,go]
+tweets:
+- "Encrypt your bits with Python 🐍 and libsodium🧂"
+- "Learn how to encrypt cross-language with Python and Go, using libsodium🧂"
+image: blog/libsodium-encryption-go-python/social.png
+type: awareness
+---
+
+The Networking and Cryptography library (NaCl pronounced "salt") is a software library that provides the core operations required to build cryptographic tools. Sodium is a fork of NaCl with an extended API; it's portable, and binaries are available to be used by various programming languages and operating systems. It comes in the form of a library called `libsodium`.
+
+Although there are several Python and Go cryptography libraries, it is primarily a matter of personal choice which one to use. One advantage of `libsodium` is that it is available for many languages.
+
+Today, we will use `libsodium` for Python to encrypt messages and decrypt them using `libsodium` in Go. Let's get started!
```

Comment:
is there any specific reason why you switched prepositions ("for Python" vs "in Go")? I would tend to keep it consistent within the sentence (for, for) or (in, in), but maybe there's a nuance i'm missing?, or maybe it was grammarly's lame suggestion...? Optional tweak. Not a hard edit.